### PR TITLE
volume-flow-imperial-gallon Added uk gallon per hour

### PR DESCRIPTION
**Pull request by aidbal**

### CODE REVIEW

File: Common/UnitDefinitions/VolumeFlow.json [hunk 1]
```diff
@@ -261,6 +261,18 @@
...
     },
+    {
+      "SingularName": "UkGallonPerHour",
+      "PluralName": "UkGallonsPerHour",
+      "FromUnitToBaseFunc": "x/791887.667",
+      "FromBaseToUnitFunc": "x*791887.667",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "gal (imp.)/hour" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/VolumeFlow.json]: `gal (imp.)/h` would be more consistent with the library. Add both or just one of them?

File: UnitsNet.Tests/CustomCode/VolumeFlowTests.cs [hunk 2]
```diff
@@ -105,6 +105,8 @@ public class VolumeFlowTests : VolumeFlowTestsBase
 
         protected override double UkGallonsPerDayInOneCubicMeterPerSecond => 1.9005304e7;
 
+        protected override double UkGallonsPerHourInOneCubicMeterPerSecond => 791888.967;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumeFlowTests.cs]: Verified.

File: Common/UnitDefinitions/VolumeFlow.json (hunk 1 above)
Comment (aidbal) [reply Common/UnitDefinitions/VolumeFlow.json]: 👍

**Comment by angularsen** [comment]
Awesome, nuget on the way out.

[Release UnitsNet/4.36.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.36.0)

**Pull request merged**